HeadsDown/ViewController.swift

```diff
@@ -41,6 +42,7 @@ class ViewController: NSViewController {
     }
     
     @IBAction func addApp(_ sender: Any) {
+        MSAnalytics.trackEvent("AddAppOpened")
```

Comment:
Maybe we can also track the interaction with the modal

1. The user added a new app
2. The user cancelled the modal
3. The user added an invalid/existing app